integration-tests/src/test/java/org/apache/fineract/integrationtests/LoanChargeSpecificDueDateTest.java

```diff
@@ -233,6 +233,70 @@ public void testApplyLoanSpecificDueDatePenaltyWithDisbursementDate() {
 
     }
 
+    @Test
+    public void testApplyAndWaiveInstallmentFee() {
```

Comment:
this test is not testing partially paid and later waived installment fee behaviour

Reply:
Thanks